apis/fluent-en_GB/atrium-api-fluent-en_GB-jvm/src/test/kotlin/ch/tutteli/atrium/api/fluent/en_GB/ChronoLocalDateTimeAssertionSpec.kt

```diff
@@ -20,6 +21,14 @@ class ChronoLocalDateTimeAssertionSpec : Spek({
         fun1(Expect<ChronoLocalDateTime<*>>::isEqual)
     )
 
+    object StringSpec : ch.tutteli.atrium.specs.integration.ChronoLocalDateTimeAssertionSpec(
+        fun1(Expect<ChronoLocalDateTime<*>>::isBefore),
```

Comment:
The goal is that we use not the function which expects a ChronoLocalDateTime but your function which expects a String. Hence create a function in the companion object with signature:
```
fun isBefore(expect: Expect<ChronoLocalDateTime<*>>, expected: ChronoLocalDateTime<*>) 
```
Which converts `expected` into a string an passes it to your string function. 
Same same but different for the other methods
```